cluster/log-dump/log-dump.sh

```diff
@@ -63,6 +63,13 @@ readonly windows_node_otherfiles="C:\\Windows\\MEMORY.dmp"
 # file descriptors for large clusters.
 readonly max_dump_processes=25
 
+# Default exit code for the log dump process.
+log_dump_exit_code=0
```

Comment:
as this is rather a marker that log exporting failed, than an exit code that is always used, could you reflect that in variable name and use?

Reply:
Makes sense. Done.